charts/incubator/reg/SCALE/questions.yaml

```diff
@@ -490,6 +490,17 @@ questions:
     schema:
       type: dict
       attrs:
+        - variable: runAsNonRoot
+          label: "runAsNonRoot"
+          schema:
+            type: boolean
+            default: true
+        - variable: runAsUser
+          label: "runAsUser"
+          description: "The UserID of the user running the application"
+          schema:
+            type: int
+            default: 568
```

Comment:
If you default this to 568, would you mind also using the same setting in values.yaml?
This ensures the test is run with runAsUser: 568 enabled ;-)